app/admin/system/lnp_resolver.rb

```diff
@@ -2,7 +2,7 @@
 
 ActiveAdmin.register System::LnpResolver do
   actions :all
-  menu parent: 'System', label: 'LNP resolvers', priority: 130
+  menu false
```

Comment:
take a look at activeadmin specs from related mr, 
it should be  menu parent: ["System", "Components"]